carma_wm_ctrl/src/WMBroadcaster.cpp

```diff
@@ -87,6 +87,7 @@ std::shared_ptr<Geofence> WMBroadcaster::geofenceFromMsg(const cav_msgs::Traffic
 
   // Get affected lanelet or areas by converting the georeference and querying the map using points in the geofence
   gf_ptr->affected_parts_ = getAffectedLaneletOrAreas(msg_v01);
+
```

Comment:
This file has a lot of changes that seem like they could be included in the Worker class so they could be unit tested.

Reply:
Nevermind. This is the worker class, just add the appropriate unit tests.